### PR TITLE
update bifrost name

### PR DESCRIPTION
**Pull request by awesomepan**

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const prodParasKusama: EndpointOption[] = [
     info: 'bifrost',
     homepage: 'https://ksm.vtoken.io/?ref=polkadotjs',
     paraId: 2001,
-    text: 'Bifrost (Kusama)',
+    text: 'Bifrost Kusama',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:52]: Any reason why this is not simply “Bifrost” (this only affects the list display). The reason for asking is that it doesn’t s already under the Kusama tab. 

(Obviously the chain name later on is 100%)

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ export const prodParasPolkadot: EndpointOption[] = [
     info: 'bifrost',
     homepage: 'https://crowdloan.bifrost.app',
     paraId: 2030,
-    text: 'Bifrost',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:57]: Same question as above ;)

Comment (awesomepan) [reply packages/apps-config/src/endpoints/productionRelayPolkadot.ts]: updated, thx.

File: packages/apps-config/src/endpoints/productionRelayKusama.ts (hunk 1 above)
Comment (awesomepan) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts]: updated thx

**Review by jacogr — APPROVED** [review]

**Pull request merged**